modules/front-end/src/app/features/safe/feature-flags/details/setting/setting.component.less

```diff
@@ -62,6 +62,28 @@
       border-radius: 8px;
     }
   }
+  .description {
```

Comment:
This change not related to the issue, it's already done. You may need to rebase your branch on to the main if this change is not done on purpose